python/seldon_core/microservice.py

```diff
@@ -342,9 +348,17 @@ def rest_prediction_server():
                     logger.info("Set JAEGER_EXTRA_TAGS %s", jaeger_extra_tags)
                     tracing = FlaskTracing(tracer, True, app, jaeger_extra_tags)
 
-                app.run(host="0.0.0.0", port=port)
+                app.run(
+                    host="0.0.0.0",
+                    port=port,
+                    threaded=False if args.single_threaded else True,
+                )
 
-            logger.info("REST microservice running on port %i", port)
+            logger.info(
```

Comment:
Related to other comment, in this case this is pure flask - no gunicorn.

Reply:
Fixed.